.env.example

```
@@ -86,6 +86,21 @@ NUXT_MCP_ADMIN_TOKEN=
+# (enable the Message Content privileged intent on the Bot tab)
+DISCORD_BOT_TOKEN=
+DISCORD_PUBLIC_KEY=
+DISCORD_APPLICATION_ID=
+# Comma-separated Discord channel ids where Nuxi answers @mentions (threads included).
+# Unset or empty means deny everywhere. Sessions from these channels are admin-enabled.
+DISCORD_ALLOWED_CHANNELS=
+# Chat SDK state (thread subscriptions, dedupe) — required in prod, memory fallback in dev
+REDIS_URL=
+# Optional override for the Gateway forward target
+# (defaults to https://$VERCEL_PROJECT_PRODUCTION_URL/eve/v1/discord)
+DISCORD_GATEWAY_WEBHOOK_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DISCORD_GATEWAY_WEBHOOK_URL key should go before the DISCORD_PUBLIC_KEY key

(UnorderedKey)